src/rgw/rgw_lc.cc

```diff
@@ -195,8 +195,10 @@ bool RGWLC::if_already_run_today(time_t& start_date)
   localtime_r(&start_date, &bdt);
 
   if (cct->_conf->rgw_lc_debug_interval > 0) {
-	  /* We're debugging, so say we can run */
-	  return false;
+    if (now - start_date < cct->_conf->rgw_lc_debug_interval)
```

Comment:
I think if you want to make "lc process" process all items of lc objects multi-times a day, you can modify this configuration item from "rgw_lc_debug_interval" to "rgw_lc_process_interval". How do you think?